stock_vlm_mgmt/models/stock_location_vlm_tray.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2023 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import _, api, fields, models
+
+
+class StockLocationVlmTray(models.Model):
+    _name = "stock.location.vlm.tray"
+    _description = "Individual trays in a Vertical Lift Module"
+
+    name = fields.Char()
```

Comment:
As @jbaudoux said, you copied this:

![image](https://github.com/OCA/stock-logistics-warehouse/assets/19529533/6f2806be-11ab-4e04-9a0a-3740264105a5)

Reply:
Yes, sorry, I was not precised enough. These pieces have been extracted respecting attribution, but the main working way is changed.